Wrappers/Python/ccpi/io/TXRMDataReader.py

```diff
@@ -0,0 +1,203 @@
+# -*- coding: utf-8 -*-
+#  CCP in Tomographic Imaging (CCPi) Core Imaging Library (CIL).
+
+#   Copyright 2019 UKRI-STFC
+#   Copyright 2019 University of Manchester
+
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+
+#   http://www.apache.org/licenses/LICENSE-2.0
+
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from ccpi.framework import AcquisitionData, AcquisitionGeometry
+import numpy
+import os
+
+import dxchange
+import olefile
+    
+        
+class TXRMDataReader(object):
+    
+    def __init__(self, 
+                 **kwargs):
+        '''
+        Constructor
+        
+        Input:
+            
+            txrm_file       full path to .txrm file
+                    
+        '''
+        
+        self.txrm_file = kwargs.get('txrm_file', None)
+        
+        if self.txrm_file is not None:
+            self.set_up(txrm_file = self.txrm_file)
+            
+    def set_up(self, 
+               txrm_file = None):
+        
+        self.txrm_file = txrm_file
+        
+        if self.txrm_file == None:
+            raise Exception('Path to txrm file is required.')
+        
+        # check if txrm file exists
+        if not(os.path.isfile(self.txrm_file)):
+            raise Exception('File\n {}\n does not exist.'.format(self.txrm_file))  
+
+    def get_geometry(self):
+        
+        '''
+        Return AcquisitionGeometry object
+        '''
+        
+        return self._ag
+        
+    def load_projections(self):
+        
+        '''
+        Load projections and return AcquisitionData container
+        '''
+        
+        # Load projections and most metadata
+        data, metadata = dxchange.read_txrm(self.txrm_file)
+        number_of_images = data.shape[0]
+        
+        # Read source to center and detector to center distances
+        ole = olefile.OleFileIO(self.txrm_file)
+        StoCdist = dxchange.reader._read_ole_arr(ole, \
```

Comment:
I think it would be cleaner if `StoCdist` would be called as the member in the file it points to, i.e.`StoRADistance`.

Also I think we follow [SIRF](https://github.com/SyneRBI/SIRF/blob/master/doc/DeveloperGuide.md#naming-)'s coding guideline and @gfardell 's [geometry ](https://github.com/vais-ral/CCPi-Framework/blob/AG_perprojection/Wrappers/Python/ccpi/framework/framework.py#L986)reworking, which would result in `dist_source_center` and `dist_center_detector`.